python/pyarrow/tests/test_compute.py

```diff
@@ -729,31 +729,6 @@ def test_generated_docstrings():
         memory_pool : pyarrow.MemoryPool, optional
             If not passed, will allocate memory from the default memory pool.
         """)
-    # Nullary with options
-    assert pc.random.__doc__ == textwrap.dedent("""\
```

Comment:
Is there a reason this was deleted instead of simply adjusted to match hypothetical docstring changes? This is used to check that we don't regress on compute docstring usability (these docstrings are generated from assorted bits of metadata in a non-trivial way).

Reply:
This function wrapper isn't auto-generated -- maybe it could be but I couldn't figure out how to make it work in the 10 or so minutes I spent on it